### DISCUSSION TITLE
Default Docker Login not working

### DISCUSSION THREAD

**Discussion by benhartwich** [directus/directus#6143] [Archive]
Hi,

I´ve installed directus via Docker 
```
docker run -d \
--name='directus' \
--restart=always \
-p 8060:8055 directus/directus
```

but the documented first user login (https://docs.directus.io/guides/installation/docker/) doesn´t work. What is the correct default user?

**Comment by rijkvanzanten** [comment]
> but the documented first user login

There is no documented first user login. It's saying that you have to provide those env vars if you want to set them to something specific, otherwise it will generate a random password for you. Check the terminal logs of the container, it should show a message saying something like "Admin password wasn't configured, generated `abcdef`"
